lib/pid/include/PID.h

```diff
@@ -204,39 +204,39 @@ class PID {
    *
    * @return The controller output as a double between outMin and outMax.
```

Comment:
```suggestion
   * @return The controller output as a float between outMin and outMax.
```

make sure to update comments when u make updates to code